tgui/packages/tgfont/README_ICON_TUTORIAL.txt

```diff
@@ -0,0 +1,26 @@
+The following is the process to implement your own icon using an svg.
```

Comment:
What everyone else has been saying, atomize this out. Means move it to a different pr